ui/pages/confirmations/components/confirm-gas-display/confirm-gas-display.test.js

```diff
@@ -50,7 +51,14 @@ const render = ({ transactionProp = {}, contextProps = {} } = {}) => {
 
 describe('ConfirmGasDisplay', () => {
   it('should match snapshot', async () => {
-    const { container } = render();
```

Comment:
previously this snapshot was not passed any transaction gasLimit

Reply:
hmm right. maybe we should revert this and update the snapshot again. Do you remember why we did this again? I feel like we were pairing

Reply:
nevermind, understood